Makefile

```diff
@@ -68,7 +68,7 @@ fix: export TARGET = dev
 fix: ## Fix linting issues in the code
 	${DOCKER_COMPOSE} run --rm app npm run lint:fix
 
-test: export TARGET = dev
+test: export TARGET = test
```

Comment:
hidden assumption that was made here: potential integration tests that use the database would be run with a separate `integration-test` Make target

Reply:
Personally I would prefer something like:

- `unit-test`: in-memory tests
- `integration-test`: tests that use the database
- `test`: `unit-test` plus `integration-test` if the former passes

But this is unrelated, would only be introduced with integration tests.